docs/Dockerfile

```diff
@@ -1,5 +1,8 @@
 # version 9.5.35
 FROM squidfunk/mkdocs-material@sha256:047452c6641137c9caa3647d050ddb7fa67b59ed48cc67ec3a4995f3d360ab32
 
+# Install Node.js
+RUN apk add --no-cache nodejs=20.15.1-r0 npm
```

Comment:
I am aware that we're running v22 elsewhere, but this is the latest version available in the image we're using and I didn't want to invest time in building it from source.

Either way this is used only for local development, while in the CI we don't use this docker file at all - so it's fine.